commit cab69e17f5dda1e5daee395c1e6bdfe593de9b9b
Author: Arnaud Morin <arnaud.morin@gmail.com>
Date:   Wed Dec 6 22:30:33 2023 +0100

    Remove Host header in nginx proxy config
    
    When nginx is proxifying the requests to OpenStack endpoints, we should not
    provide any Host header, this may prevent the upstream server to respond
    correctly on the proxyfied URL.
    
    Change-Id: I091f58b1906607f892a504df485a5da2dabacff5

diff --git a/skyline_apiserver/templates/nginx.conf.j2 b/skyline_apiserver/templates/nginx.conf.j2
index 29c9358..19655ed 100644
--- a/skyline_apiserver/templates/nginx.conf.j2
+++ b/skyline_apiserver/templates/nginx.conf.j2
@@ -117,7 +117,6 @@ http {
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
             proxy_set_header X-Forwarded-Host $host;
-            proxy_set_header Host $http_host;
         }
         {% endfor %}
     }
